README.md

```diff
@@ -944,6 +1019,58 @@ For the example above, this path will resolve to `/terraform-code/frontend-app/.
 what you want.
 
 
+#### get_parent_dir
```

Comment:
Ah, this is a nice helper to have for sure. Perhaps a clearer and more consistent name would be `get_parent_tfvars_dir`?